### PR TITLE
LWG Motion 19: P1902R1 Missing feature-test macros 2017-2019

### LINKED ISSUE #3421: [2019-11 LWG Motion 19] P1902R1 Missing feature-test macros 2017-2019
This resolves NB comments FI015, GB146, US150, US167, DE168,

### PR DESCRIPTION
**Pull request by burblebee**
Fixes NB FI 15, GB 146, US 150, US 167, and DE 168 (C++20 CD), and LWG2334.

Changes to macros in table 36 of [support.limits.general] were
applied to the macro definitions in [version.syn].

Fixes #3421.
Fixes cplusplus/nbballot#15
Fixes cplusplus/nbballot#145
Fixes cplusplus/nbballot#146
Fixes cplusplus/nbballot#149
Fixes cplusplus/nbballot#165
Fixes cplusplus/nbballot#166

Fixes cplusplus/papers#650

### CODE REVIEW

File: source/support.tex [hunk 1]
```diff
@@ -571,17 +574,20 @@
 #define @\defnlibxname{cpp_lib_byte}@                              201603L // also in \libheader{cstddef}
 #define @\defnlibxname{cpp_lib_char8_t}@                           201907L
   // also in \libheader{atomic}, \libheader{filesystem}, \libheader{istream}, \libheader{limits}, \libheader{locale}, \libheader{ostream}, \libheader{string}, \libheader{string_view}
-#define @\defnlibxname{cpp_lib_chrono}@                            201907L // also in \libheader{chrono}
+#define @\defnlibxname{cpp_lib_chrono}@                            201803L  // also in \libheader{chrono}
```
Comment (johelegp) [inline source/support.tex]:
```suggestion
#define @\defnlibxname{cpp_lib_chrono}@                            201803L // also in \libheader{chrono}
```

Comment (cpplearner) [inline source/support.tex]: Does P1902R1 really mean to bump *down* the value of `__cpp_lib_chrono`?

Comment (timsong-cpp) [reply source/support.tex]: @BRevzin

Comment (BRevzin) [reply source/support.tex]: @burblebee Gah! This is my mistake. R0 of the paper introduced a new macro for the date library and the suggestion was to bump the preexisting chrono macro instead. 

When I changed it to bump the old macro, I didn't look at what the current value was.

The intent was an intermediate bump, so no change in value overall 

My mistake and good catch!

**Comment by jensmaurer** [comment]
This also fixes GB 147. Please fix the commit comment accordingly.

File: source/support.tex (hunk 1 above)
Comment (burblebee) [reply source/support.tex]: Ok, reverted back to 201907L.  @brevzin If this was the wrong thing to do, please unresolve this.  Thanks!

Comment (BRevzin) [reply source/support.tex]: @burblebee Looks good to me!

**Comment by zygoloid** [comment]
(We don't need the [needs rebase] label on motion pull requests; they get rebased as part of the merge process, and we want them to be based on the pre-motions master before that.)

**Pull request merged**